lib/api/multiObjectDelete.js

```diff
@@ -50,18 +53,27 @@ import { isBucketAuthorized } from './apiUtils/authorization/aclChecks';
 * @param {boolean} quietSetting - true if xml should just include error list
 * and false if should include deleted list and error list
 * @param {object []} errorResults - list of error result objects with each
-* object containing -- key: objectName, error: arsenal error
-* @param {string []} deleted - list of object keys deleted
+* object containing -- entry: { key, versionId }, error: arsenal error
+* @param {object []} deleted - list of object deleted, an object has the format
+*     object: { entry, result, isDeletingDeleteMarker }
+*     object.entry : above
+*     object.result: stringification of { versionId }
```

Comment:
this is changed since we don't seem to need the information inside result, just whether a delete marker was created.
before, it was the parsed but unencrypted version id (metadata returns as JSON object `{ versionId }`, we parse this in the function we get 'result' from) if a delete marker was created; undefined if an object was deleted